odl/phantom/noise.py

```diff
@@ -67,6 +67,58 @@ def white_noise(space, mean=0, stddev=1, seed=None):
     return space.element(values)
 
 
+def uniform_noise(space, low=0, high=1, seed=None):
+    """Uniformly distributed noise in ``space``, pointwise ``U(low, high)``.
+
+    Parameters
+    ----------
+    space : `FnBase` or `ProductSpace`
+        The space in which the noise is created.
+    low : ``space.field`` element or ``space`` `element-like`, optional
+        The lower bound of the uniform noise. If a scalar, it is interpreted as
+        ``low * space.one()``.
+        If ``space`` is complex, the real and imaginary parts are interpreted
+        as the mean of their respective part of the noise.
```

Comment:
Hm, I don't really understand this. From the code I think it's just taking real and imaginary parts separately, no?

Reply:
"mean" is a doc copy paste from white noise.